### DISCUSSION TITLE
Enable public comments on posts

### DISCUSSION THREAD

**Discussion by byorn** [avitorio/outstatic#228] [Ideas]
Hi Guys,

Kudos to you @avitorio for this awesome project.

Can I build a feature to accept public comments for a blog post ? 

The comments can be stored via 

1) A configurable back end url, for any one who uses outstatic can host their own comments
2) Store comments within github itself itself ? might not be a good idea.

I would like to contribute to open source projects and since I host my blog http://www.byorns-playground.com I thought it would be great to add value to this project.

Please advise on how I could contribute to this project further ?

cheers,
Byorn

**Comment by avitorio** [comment]
Hey @byorn - thanks for thr suggestion.

The ability to add comments is something we want to include in the framework eventually. But this can be achieved today using third party tools.

I agree with you that storing comments in the repository doesn't sound like the right approach.

Perhaps we could add https://giscus.app to the advanced template.

**Reply by srelbo** [reply]
Or perhaps live blocks integration? -- https://liveblocks.io/
